doculog/git.py

```diff
@@ -12,25 +12,16 @@
 
 def _get_git_command() -> str:
     if sys.platform.startswith("win"):
-        path = os.environ.get("PATH", "").split(os.pathsep)
-        path_exts = os.environ.get("PATHEXT", ".exe;.bat;.cmd").split(";")
-        has_ext = os.path.splitext("git")[1] in path_exts
-        if not has_ext:
-            exts = path_exts
-        else:
-            # Don't try to append any extensions
-            exts = [""]
-
-        for d in path:
-            try:
-                for ext in exts:
-                    exepath = os.path.join(d, "git" + ext)
-                    if os.access(exepath, os.X_OK):
-                        return exepath
-            except OSError:
-                pass
-
-        return "git.cmd"
+        git_paths = subprocess.check_output(["where", "git"]).decode(sys.stdout.encoding).split('\n')
+        try:
+            for exepath in git_paths:
+                exepath = exepath.strip()
+                if os.access(exepath, os.X_OK):
+                    return exepath
+        except OSError:
+            pass
+
+            return "git.cmd"
```

Comment:
This should be un-nested from `pass`, right?

Reply:
Yeah, my bad there.